lib/supplejack_common/enrichments/abstract_enrichment.rb

```diff
@@ -21,11 +21,11 @@ def initialize(name, options, record, parser_class)
     end
 
     def primary
-      @primary ||= SupplejackCommon::FragmentWrap.new(record.fragments.where(priority: 0).first)
+      @primary ||= SupplejackCommon::FragmentWrap.new(record.fragments.select { |f| f.priority.zero? }.first)
```

Comment:
Use find instead of select.first :)

Reply:
We avoided using find because the old code was written with in a way that only takes the first of possibly many results.  find will return many results potentially, and we would need to call.first on it too.